index.bs

```diff
@@ -228,11 +238,13 @@ When an {{XRSession}} is created with depth sensing enabled, the {{XRSession/dep
 In order to <dfn>find supported configuration combination</dfn> for depth sensing API given |usagePreference| and |dataFormatPreference| sequences, the user agent MUST run the following algorithm:
 
   1. Let |selectedUsage| be <code>null</code>.
+  1. If the |usagePreference| sequence is empty, set |selectedUsage| to the device's preferred [=native depth sensing=] capability.
```

Comment:
NIT: Would it be more clear to make this and the following step an If/Else or If/Otherwise type block? (Not sure if bikeshed supports that).

I guess functionally it's the same because you'd not have anything to run through for the next step; but it may be a little cleaner.

Same for the description of the |dataFormatPreference|